### PR TITLE
Adding Purchase Events page

### PR DESCRIPTION
**Pull request by bre-fitzgerald**
# Pull Request/Issue Resolution

**Description of Change:**
> I'm adding documentation on Purchase Events and Purchase Properties

Closes #**BD-840**, **BD-619**

### Is this change associated with a Braze feature/product release?
- [ ] Yes (__Insert Feature Release Date Here__)
- [x] No

> If yes, please note the date of the feature release.

---
---

<details>
<summary>PR Checklist</summary>

- [ ] Check that all links work!
- [ ] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is _done and ready to be reviewed for merge_.
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

</details>

<details>
<summary>Writing & Style Quick Tips</summary>

- Use the formatting shown on the [Styling Test Page](https://github.com/Appboy/braze-docs/blob/develop/_docs/_home/styling_test_page.md) to create unique templates and vehicles for your information.
- Use Action verbs!
- Capitalize proper nouns, unique Braze tools and features, custom Braze channels, and all Header titles.
- Redirect when necessary!
- Always speak to the Braze customer - refer to them as “you”, whether they are an Engineer/Developer, a Marketer, or an Admin.
- On Voice: You should aim to be a Friendly Professor. Imagine, if you will, the smartest, but also kindest professor you had. You know, the one who showed you your potential and tested you, but also went slowly in class so you could take great notes and learn some things? Don't be too peppy, but don't be formal. You’re not selling anything - you’re instructing and informing and enabling.

_Consult the [Docs Text Formatting Guide](https://github.com/Appboy/braze-docs/wiki/Special-Formatting) and the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/braze-docs/wiki/Writing-Style-Guide-&-Best-Practices) for more details._

</details>

<details>
<summary>ATTENTION: For PR Reviewers</summary>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
</details>

---
---

### CODE REVIEW

File: _docs/_api/objects_filters/purchase_object.md [hunk 1]
```diff
@@ -56,10 +56,12 @@ Within the purchase object, The `product_id` is an identifier for the purchase,
...
+
+This helps make products easy to identify for segmentation and triggering.
 
 ## Purchase Properties Object
-Custom events and purchases may have event properties. The “properties” values should be an object where the keys are the property names and the values are the property values. Property names must be non-empty strings less than or equal to 255 characters, with no leading dollar signs. Property values can be integers, floats, booleans, datetimes (as strings in ISO8601 or yyyy-MM-dd'T'HH:mm:ss:SSSZ format), or strings less than or equal to 255 characters.
+Custom events and purchases may have event properties. The “properties” values should be an object where the keys are the property names and the values are the property values. Property names must be non-empty strings less than or equal to 255 characters, with no leading dollar signs. Property values can be integers, floats, booleans, datetimes (as strings in [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) or yyyy-MM-dd'T'HH:mm:ss:SSSZ format), or strings less than or equal to 255 characters.
 
 ### Purchase Properties
```
Comment (KellieHawks) [inline _docs/_api/objects_filters/purchase_object.md:66]: Can you link out to the purchase properties section here?

File: _docs/_user_guide/data_and_analytics/custom_data/purchase_events.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
+---
+nav_title: Purchase Events
```
Comment (KellieHawks) [inline _docs/_user_guide/data_and_analytics/custom_data/purchase_events.md:2]: Can you add this page to the custom data landing page?

**Review by KellieHawks — CHANGES_REQUESTED** [review]
Looks great! Two minor edits requested. Thanks!

File: _docs/_user_guide/data_and_analytics/custom_data/purchase_events.md (hunk 2 above)
Comment (bre-fitzgerald) [reply _docs/_user_guide/data_and_analytics/custom_data/purchase_events.md:2]: Done!

File: _docs/_api/objects_filters/purchase_object.md (hunk 1 above)
Comment (bre-fitzgerald) [reply _docs/_api/objects_filters/purchase_object.md:66]: Added!

**Review by KellieHawks — APPROVED** [review]
Looks great! Thanks for making those changes.

**Pull request merged**